pyresample/test/test_bucket.py

```diff
@@ -216,13 +217,31 @@ def test_get_max(self):
         data = da.from_array(np.array([[2, 11], [5, np.nan]]),
                              chunks=self.chunks)
         result = self._get_max_result(data)
-        # test multiple entries average
+        # test multiple entries maximum
         self.assertEqual(np.count_nonzero(result == 11), 1)
-        # test single entry average
+        # test single entry maximum
         self.assertEqual(np.count_nonzero(result == 5), 1)
         # test that minimum of bucket with only nan is nan, and empty buckets are nan
         self.assertEqual(np.count_nonzero(~np.isnan(result)), 2)
 
+    def test_get_abs_max(self):
```

Comment:
Instead of counting how many elements equal -11/5 etc, would it be possible to compare the entire result array with the expectation?

Reply:
:smile: Ok I see